webapp/src/components/centerPanel.tsx

```diff
@@ -112,14 +135,55 @@ class CenterPanel extends React.Component<Props, State> {
         return true
     }
 
+    shouldStartBoardsTour(): boolean {
```

Comment:
nit: This component is already big enough, can we move this logic into something else, maybe a "behavior component", something like a component called "BoardsTourInit" that takes care of this with the lifecycle, that way you can simplify a lot that and remove from this already big component all this logic.

Reply:
This is not a generic logic but rather only specific for the board tour being displayed in this component. So won't make sense storing it outside.